src/profile-logic/transforms.js

```diff
@@ -256,6 +273,8 @@ export function stringifyTransforms(transformStack: TransformStack): string {
       // need only a funcIndex, while some care about the current implemention, or
       // other pieces of data.
       switch (transform.type) {
+        case 'merge-function-set':
+          return `${shortKey}-${[...transform.funcIndexes].join('-')}`;
```

Comment:
Hmm, if we expect hundreds of elements in this array, would it make sense to use `encodeUintArrayForUrlComponent` for this?

Reply:
It would be good to do it before landing, so that we don't have to write an url upgrader :-)

Reply:
Ah I forgot about this thanks.